src/api.ts

```diff
@@ -0,0 +1,65 @@
+export interface Config {
+    url: string | URL;
+    jwt?: string;
```

Comment:
Exactly my thoughts! What do you think about using `token`? We already use `jwt` in some places (`@libsql/sqlite3`, the Go client library, the Hrana protocol), but we still have time to change that.

Reply:
I thought about just "token", but then realized that there could conceivably be more than one type of token out there.  authToken makes it clear what sort of token we're talking about, and we can underscore that in the CLI and docs as well.

Reply:
Hmm, what kind of token could we conceivably use? If the parameter is passed by URL, we should keep the name as short as possible.

Also, using two words creates a problem with the casing convention: should we use `authToken`, `auth_token`, `auth-token`, `authtoken`? The `Config` object should follow the language conventions, so the token should be `authToken` in JS and `auth_token` in Rust. But the URLs should be the same in all languages, so the URL parameter is bound to be inconsistent with the `Config` field in some languages (`authToken` in URL, `auth_token` in `Config` in Rust).

Reply:
I'm not concerned about the casing convention - we can do whatever is normal for the language.  I'm just concerned about misinterpretation of "token" going forward because the word has many different definitions.

https://en.wikipedia.org/wiki/Token

Reply:
Ok, changed.